src/applications/edu-benefits/10203/containers/IntroductionPage.jsx

```diff
@@ -49,10 +23,7 @@ export class IntroductionPage extends React.Component {
 
   loginPrompt() {
     if (this.props.isLoggedIn) {
-      if (
-        this.props.useEvss &&
```

Comment:
I removed this because 
1. We aren't using EVSS anymore and this is confusing and
2. The prop was constructed from a feature flag which no longer exists (so was always false)